au4/src/effects/effects_base/internal/effectsactionscontroller.cpp

```diff
@@ -22,5 +26,24 @@ void EffectsActionsController::doEffect(const muse::actions::ActionData& args)
 
     muse::String effectId = args.arg<muse::String>(0);
 
-    effectExecutionScenarion()->performEffect(effectId);
+    effectExecutionScenario()->performEffect(effectId);
+}
+
+void EffectsActionsController::repeatLastEffect(const muse::actions::ActionData&)
```

Comment:
By the way, if the action has no arguments, then you don’t have to specify them

Reply:
Of course ...